lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss

```diff
@@ -130,3 +130,81 @@
   }
 
 }
+
+.campaign--pitch {
+  .cta.-persistent {
+    display: none;
+  }
+
+  &.-persistent-beta {
+    .cta {
+      display: none;
+    }
+
+    .header__signup {
+      display: none;
+    }
+
+    .cta.-persistent {
+      display: block;
+      background-color: $black;
+      border: none;
+      position: sticky;
+      top: 0;
+      z-index: 100;
+
+      // Fallback for position:sticky, through Filament Group's fixed-sticky
+      &.fixedsticky-on {
+        position: fixed;
+        width: 100%;
+        max-width: 1440px;
+      }
+
+      > .wrapper {
+        @include clearfix;
+        display: table;
+        padding: $base-spacing / 2 0px;
+      }
+
+      .cta__button {
+        display: table-cell;
+        @include span(5 of 12);
+        float: none;
+        padding: 0px;
+        text-align: left;
+
+        @include media($medium) {
+          width: auto;
+          float: left;
+          display: inline-block;
+        }
+      }
+
+      .cta__message {
+        display: table-cell;
+        @include span(7 of 12);
```

Comment:
Let's move this mix-in to the top of the rule, so it's clear `table-cell` still gets applied & isn't being overridden.